app/controllers/events_controller.rb

```diff
@@ -11,17 +10,15 @@ def show
   def new
     @event = Event.new
   end
-  
-  def  create
-    
+
+  def create
     @event = current_user.events.build(event_params)
     @event.creator_id = session[:user_id]
     if @event.save
-      flash[:success] = "Event successfully created"
+      flash[:success] = 'Event successfully created'
       redirect_to event_path(@event)
     else
-
-      flash[:error] = "Something went wrong"
+      flash[:error] = 'Something went wrong'
```

Comment:
## Error creating events
- [ ] We have an issue creating events. Kindly revisit your migrations and how you are defining methods on this file to track down this bug 👇 Thank you.
![Screenshot 2020-09-28 211613](https://user-images.githubusercontent.com/22345961/94470497-6f79a300-01d0-11eb-81fa-a44bc90e8a28.png)